tests/topotests/bgp_vpnv4_ebgp_vpn_auto/r1/bgpd.conf

```diff
@@ -0,0 +1,71 @@
+bgp route-map delay-timer 1
+router bgp 65500
+ bgp router-id 192.0.2.1
+ no bgp ebgp-requires-policy
+ neighbor 192.168.0.2 remote-as 65501
+ address-family ipv4 unicast
+  no neighbor 192.168.0.2 activate
+ exit-address-family
+ address-family ipv4 vpn
+  neighbor 192.168.0.2 activate
+ exit-address-family
+!
```

Comment:
please reformulate commit log: title + detail

topotets: add test to control release of bgp labels